SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -200,6 +204,8 @@ void ASpatialMetricsDisplay::Tick(float DeltaSeconds)
 	FWorkerStats Stats{};
 	Stats.WorkerName = SpatialNetDriver->Connection->GetWorkerId().Left(WorkerNameMaxLength).ToLower();
 	Stats.AverageFPS = Metrics.GetAverageFPS();
+	Stats.ServerConsiderListSize = SpatialNetDriver->GetConsiderListSize();
```

Comment:
Is there a `#if !UE_BUILD_SHIPPING` around this? I can't seem to find it.

Reply:
There is not - good catch